cross/nettle/Makefile

```diff
@@ -32,7 +32,7 @@ endif
 ifeq ($(findstring $(ARCH), $(x64_ARCHES)),$(ARCH))
 CONFIGURE_ARGS = --target=x86_64-pc-linux-gnu --host=x86_64-pc-linux-gnu
 endif
-CONFIGURE_ARGS += --prefix=$(INSTALL_PREFIX) --enable-shared
+CONFIGURE_ARGS += --prefix=$(INSTALL_PREFIX) --enable-shared --libdir=$(INSTALL_PREFIX)/lib
```

Comment:
As far as I have tested with `arch-x64-6.1` on my Debian system, this option is not required at all as it is default behaviour. May you please check again or report if it is for only specific architecture.

Reply:
My finding was that using specific 64-bit architectures (e.g. `make arch-apollolake`) would put some of the libraries under `lib64` instead of `lib`.  Thus other packages depending of it could not find them afterwards.  I would guess it is specific to theses sub-architectures?  For sure, this was mandatory for apollolake.

Reply:
OK. Strange. To be honest, synocommunity packages are built with generic `arch-x64-6.1` target which generates a single binary package for all intel 64 bits architectures.

Reply:
I'll try to find some time and confirm which of the 64-bit sub-arch are affected other than apollolake.  A `ifeq` statement could be added on a sub-arch basis type-thing...

Reply:
I can already tell you this option is not required for `arch-x64-6.1` (generic intel 64 bits), `arch-aarch64-6.1` (ARM v8 64 bits), `arch-evansport-6.1` and `arch-apollolake-6.1`; neither in Docker context, nor in my "native" up-to-date Debian testing 64 bits with multi arch support (32 bits enabled). I really do not understand what may have triggered such a "failure" in your context/system.

Reply:
That's rather interesting as I'm not getting the same results...
So lets trace back:
1) default minimal install of debian + setup `sudo`, ssh and `locales`
2) installations of the requirements (as per patch 01)
3) `sudo pip install -U setuptools pip wheel httpie`
4) then `spkrc`
```
$ git clone https://github.com/SynoCommunity/spksrc.git
$ cd spksrc
$ make setup
Setting default toolchain version to DSM-6.1
$ cd cross/nettle
$ make arch-apollolake
$ make arch-avoton
$ make arch-denverton
$ make arch-grantley
$ make arch-rtd1296
$ make arch-x64
```
Now checking the end-results:
```
$ ls -1ad work-*
work-apollolake-6.1
work-avoton-6.1
work-denverton-6.1
work-grantley-6.1
work-rtd1296-6.1
work-x64-6.1
```
And who used `lib64`:
```
$ ls -1ad work-*/install/usr/local/nettle/lib64/libnettle.so
work-apollolake-6.1/install/usr/local/nettle/lib64/libnettle.so
work-avoton-6.1/install/usr/local/nettle/lib64/libnettle.so
work-denverton-6.1/install/usr/local/nettle/lib64/libnettle.so
work-grantley-6.1/install/usr/local/nettle/lib64/libnettle.so
work-x64-6.1/install/usr/local/nettle/lib64/libnettle.so
```
Only `rtd1296` stayed into `lib` without any offending usage of `lib64`.

Reply:
Was someone else able to reproduce?
Is there anything else I can do to get merged upstream?